plasmapy/diagnostics/tests/test_langmuir.py

```diff
@@ -198,7 +198,7 @@ class DryCharacteristic(langmuir.Characteristic):
     unique values.
     """
 
-    def __init__(self, bias, current):
+    def __init__(self, bias, current) -> None:
```

Comment:
```suggestion
    def __init__(self, bias, current):
```
...since it's covered in another PR.